build.rs

```diff
@@ -0,0 +1,14 @@
+extern crate protoc_rust;
+
+fn main() {
+    if cfg!(feature = "gen") {
+        protoc_rust::run(protoc_rust::Args {
+            out_dir: "proto",
+            input: &["proto/metrics.proto"],
+            includes: &["proto"],
+            customize: protoc_rust::Customize {
```

Comment:
can we use `protoc_rust::Customize::default()` here?

Reply:
sure, fixed, just copy from `rust-protobuf` example code